src/Kiota.Builder/Writers/HTTP/CodePropertyWriter.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Kiota.Builder.CodeDOM;
+using Kiota.Builder.Extensions;
+
+namespace Kiota.Builder.Writers.Http;
+public class CodePropertyWriter(HttpConventionService conventionService) : BaseElementWriter<CodeProperty, HttpConventionService>(conventionService)
+{
+    public override void WriteCodeElement(CodeProperty codeElement, LanguageWriter writer)
```

Comment:
I removed all non-RequestBuilder types from the generated code DOM. However, the remaining RequestBuilder classes can still contain properties, functions, and other elements. If I were to remove the writer stubs entirely, it would result in an exception when the writer encounters these elements and attempts to delegate their writing to the language-specific writers, which would be null in this case.